src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -121,8 +121,6 @@ def get_test_file_path(test_name, file_name):
 
 @pytest.fixture(params=[array.array, numpy.array, list])
 def source_waveform_type(request):
-    if request.param != array.array:
-        pytest.skip('Source waveform types other than array.array are not supported yet (#1132)')
     return request.param
```

Comment:
do we have tests with the wrong type of inputs, to ensure the checks work?

Reply:
Created an issue - #1185